.github/workflows/ci.yml

```diff
@@ -31,6 +31,8 @@ jobs:
         with:
           path: .cache/bazel-repo
           key: bazel-repo
+      - name: Validate changed files
+        run: ./.github/workflows/validateChangedFiles.sh
```

Comment:
The rest of the project uses snake-case whenever possible instead of camel-case.